app/javascript/pages/MyEvents/DeleteModalContents.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react'
+import { Header, Body, Footer, FooterItem } from '@zendeskgarden/react-modals'
+import { Dots } from '@zendeskgarden/react-loaders'
+import { Button } from '@zendeskgarden/react-buttons'
+
+const DeleteModalContents = ({ toDelete, deleteIndividualEvent, setShowDeleteModal }) => {
+  const [loading, setLoading] = useState(false)
+
+  const handleDelete = () => {
+    setLoading(true)
+    deleteIndividualEvent(toDelete.id).then(() => {
+      setLoading(false)
+      setShowDeleteModal(false)
+    })
+  }
+
+  return (
+    <>
+      <Header isDanger>Delete Event</Header>
```

Comment:
Should this be danger? usually we use danger to represent an error. This is more of a warning imo.

Reply:
I think danger is fine to represent a dangerous action (such as deletion)

Reply:
true